kube_watcher/kube_watcher.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016 Intel Corporation
```

Comment:
A great canonical way to write this would to make it independent of the drainer - it should simply be a watcher - propagating the events upwards into a channel. The logic of the actions should then be taken in the actual drainer code. This package should ideally not call any external functions - to make it a re-usable minimal library.